external/js/cothority/src/skipchain/skipchain-rpc.ts

```diff
@@ -180,15 +180,16 @@ export default class SkipchainRPC {
             const last = newBlocks[newBlocks.length - 1];
             let isInRoster = false;
             for (const n of last.roster.list) {
-                if (n.getWebSocketAddress() === this.conn.getURL()) {
+                if (n.getWebSocketAddress().replace(/^ws+:\/\//, "") ===
+                    this.conn.getURL().replace(/^ws+:\/\//, "")) {
```

Comment:
Why is it needed?

Reply:
When running the webapp locally, but connected to the real byzcoin. It's a 'manual testing' usecase, but I still do it a lot...

Reply:
I don't get it, why the URL would be different? `getWebSocketAddress` is used when creating `this.conn` so it should be the same

Reply:
if the URL is `https://`, `getWebSocketAddress` will return `wss://`, while the conneted URL returns `ws://`. So it will not match.

Reply:
Ho, right, the weird secure context.